<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SWMainUI</class>
 <widget class="QMainWindow" name="SWMainUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>640</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>QSW - Qt SpellWork for 1.12.x</string>
  </property>
  <property name="windowIcon">
   <iconset resource="SpellWork.qrc">
    <normaloff>:/SpellWork/Recources/mangos.ico</normaloff>:/SpellWork/Recources/mangos.ico</iconset>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <property name="margin">
     <number>5</number>
    </property>
    <property name="spacing">
     <number>5</number>
    </property>
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>280</width>
        <height>130</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>280</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Spell Filter</string>
      </property>
      <widget class="QFontComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>4</x>
         <y>15</y>
         <width>271</width>
         <height>20</height>
        </rect>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="currentIndex">
        <number>184</number>
       </property>
       <property name="maxVisibleItems">
        <number>20</number>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="currentFont">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>8</pointsize>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
      </widget>
      <widget class="QFontComboBox" name="comboBox_2">
       <property name="geometry">
        <rect>
         <x>4</x>
         <y>36</y>
         <width>271</width>
         <height>20</height>
        </rect>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="currentIndex">
        <number>184</number>
       </property>
       <property name="maxVisibleItems">
        <number>20</number>
       </property>
       <property name="currentFont">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>8</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QFontComboBox" name="comboBox_3">
       <property name="geometry">
        <rect>
         <x>4</x>
         <y>57</y>
         <width>271</width>
         <height>20</height>
        </rect>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="currentIndex">
        <number>184</number>
       </property>
       <property name="maxVisibleItems">
        <number>20</number>
       </property>
       <property name="currentFont">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>8</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QFontComboBox" name="comboBox_4">
       <property name="geometry">
        <rect>
         <x>4</x>
         <y>78</y>
         <width>271</width>
         <height>20</height>
        </rect>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="currentIndex">
        <number>184</number>
       </property>
       <property name="maxVisibleItems">
        <number>20</number>
       </property>
       <property name="currentFont">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>8</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QFontComboBox" name="comboBox_5">
       <property name="geometry">
        <rect>
         <x>4</x>
         <y>99</y>
         <width>271</width>
         <height>20</height>
        </rect>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="currentIndex">
        <number>184</number>
       </property>
       <property name="maxVisibleItems">
        <number>20</number>
       </property>
       <property name="currentFont">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>8</pointsize>
        </font>
       </property>
      </widget>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>280</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>280</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Advanced Filter</string>
      </property>
      <widget class="QLineEdit" name="adLine1">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>20</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="adLine2">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>43</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QFontComboBox" name="adBox1">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>20</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="currentIndex">
        <number>184</number>
       </property>
       <property name="maxVisibleItems">
        <number>20</number>
       </property>
       <property name="currentFont">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>8</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QFontComboBox" name="adBox2">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>43</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="currentIndex">
        <number>184</number>
       </property>
       <property name="maxVisibleItems">
        <number>20</number>
       </property>
       <property name="currentFont">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>8</pointsize>
        </font>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="0" rowspan="4">
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="opaqueResize">
       <bool>true</bool>
      </property>
      <property name="handleWidth">
       <number>5</number>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QGridLayout" name="gridLayout" rowstretch="0,0">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>6</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="slabel1">
          <property name="text">
           <string>Spell Id 1:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="compareSpell_1"/>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QWebView" name="webView">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(206, 206, 206);</string>
          </property>
          <property name="url">
           <url>
            <string>about:blank</string>
           </url>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget1">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="slabel2">
          <property name="text">
           <string>Spell Id 2:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="compareSpell_2"/>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QWebView" name="webView_2">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(206, 206, 206);</string>
          </property>
          <property name="url">
           <url>
            <string>about:blank</string>
           </url>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>280</width>
        <height>110</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>280</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Spell Search</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <widget class="QLabel" name="findLine_l1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>23</y>
         <width>58</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ID or Name:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="findLine_e1">
       <property name="geometry">
        <rect>
         <x>74</x>
         <y>22</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="cursorPosition">
        <number>0</number>
       </property>
      </widget>
      <widget class="QLabel" name="findLine_l2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Icon ID:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="findLine_e2">
       <property name="geometry">
        <rect>
         <x>74</x>
         <y>80</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="findLine_l3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Description:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="findLine_e3">
       <property name="geometry">
        <rect>
         <x>74</x>
         <y>50</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QTableView" name="SpellList">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>280</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="autoScroll">
       <bool>true</bool>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="tabKeyNavigation">
       <bool>true</bool>
      </property>
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideMiddle</enum>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>40</number>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>17</number>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>17</number>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionExit">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="modeStandart">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Standart</string>
   </property>
  </action>
  <action name="modeCompare">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Compare</string>
   </property>
  </action>
  <action name="actionRegExp">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Use RegExp</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="SpellWork.qrc"/>
 </resources>
 <connections/>
</ui>
